### PR TITLE
stm32flash 0.5 (new formula)

### PR DESCRIPTION
**Pull request by rickard-von-essen**
Open source flash program for STM32 using the ST serial bootloader.
See https://sourceforge.net/projects/stm32flash/

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/stm32flash.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  homepage "https://sourceforge.net/projects/stm32flash/"
+  url "https://downloads.sourceforge.net/project/stm32flash/stm32flash-0.5.tar.gz"
+  sha256 "97aa9422ef02e82f7da9039329e21a437decf972cb3919ad817f70ac9a49e306"
+
+  def install
+    system "make", "PREFIX=#{prefix}", "install"
+  end
+
+  test do
+    system "#{bin}/stm32flash", "-h"
```
Comment (fxcoudert) [inline Formula/stm32flash.rb]: We need a test that exercises the some of the functionality of the app.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/stm32flash.rb (hunk 1 above)
Comment (rickard-von-essen) [reply Formula/stm32flash.rb]: Unfortunately there is nothing else you can do without a USB to UART converter and a compatible device connected.

Comment (fxcoudert) [reply Formula/stm32flash.rb]: Can we try to perform an operation, and match the failure (exit code and error message)?

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @rickard-von-essen for the pull request!

**Pull request closed without merging**